helm/operator/values.yaml

```diff
@@ -183,6 +183,24 @@ dataLoadInit:
   imageRepository: hwameistor/dataload-init
   tag: v0.0.1
 
+# StorageClass Settings
+storageClass:
+  # If enabled, a hwameistor storageclass will be created in your system
+  enable: true
+  # If enabled, hwameistor storageclass will be a default storageclass in system
+  # Attention: ensure your system doesn't have any default storageclass before
+  # more info: https://kubernetes.io/docs/tasks/administer-cluster/change-default-storage-class/
+  default: false
```

Comment:
do operator use `enable` and `default` options now? if not, suggest remove it from values.yaml untill we support indeed.